### ISSUE TITLE
bitmap error on draw

### ISSUE THREAD

**Issue by kuldeep8486** [wooplr/Spotlight#66]
In your library you have mentioned that you have put try catch for onDraw method (in issue #26 ), but when importing it from gradle I can't see it.
![screenshot 34](https://user-images.githubusercontent.com/18348756/28454127-b680eaf6-6e17-11e7-952d-4f5f16f789c2.png)

**Comment by dhavalwooplr** [comment]
@kuldeep8486 , Try-catch is there. I checked code when last release was done and it was there in that code. But somehow, if you didn't get it via update then you can always download a fresh code from repo and add it as a module in your project.

**Comment by kuldeep8486** [comment]
Yes, did the same thing, thanks for the reply and a nice library to save our time.
